applicationset/utils/utils_test.go

```diff
@@ -1185,6 +1185,43 @@ func TestNormalizeBitbucketBasePath(t *testing.T) {
 	}
 }
 
+func TestSlugify(t *testing.T) {
```

Comment:
Would you mind also adding a test that demonstrates this function works in an AppSet template?